test/e2e/specs/toppanel.spec.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+ 'Basic top panel test': function (client){
+   var data = client.globals;
+   client
+    .url(client.launch_url)
+    .click('.toppanel-hide')
+    .waitForElementNotVisible('.toppanel-hide')
```

Comment:
Can you assert the element is visible?
Is it async? Do you have to actually wait for it?